packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md

```diff
@@ -32,29 +32,23 @@ Authorization: SSWS 00QCjAl4MlV-WPXM...0HmjFx-vbGua
 
 ### Privilege level
 
-Different Okta API operations require different admin privilege levels. API tokens inherit the privilege level of the admin account that is used to create them. It is therefore good practice to create a service account to use when you create API tokens so that you can assign the token the specific privilege level needed. See [Administrators](https://help.okta.com/okta_help.htm?id=ext_Security_Administrators) for admin account types and the specific privileges of each.
+Different Okta API operations require different admin privilege levels. API tokens inherit the privilege level of the admin account that is used to create them. It’s therefore good practice to create a service account to use when you create API tokens so that you can assign the token the specific privilege level needed. See [Administrators](https://help.okta.com/okta_help.htm?id=ext_Security_Administrators) for admin account types and the specific privileges of each.
 
 ### OAuth 2.0 instead of API tokens
 
-As an alternative to Okta API tokens, you can now interact with Okta APIs using scoped OAuth 2.0 access tokens for a number of Okta endpoints. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains. For more information, see the [OAuth for Okta](/docs/guides/implement-oauth-for-okta/) guide.
+As an alternative to Okta API tokens, you can now interact with Okta APIs. You can use a scoped OAuth 2.0 access token for various Okta endpoints. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains. For more information, see the [OAuth for Okta](/docs/guides/implement-oauth-for-okta/) guide.
 
 ## Create the token
 
-To create an API token, follow these steps:
+See [Create Okta API tokens](https://help.okta.com/okta_help.htm?id=ext-create-api-token) to create an API token and define the network zones that API calls can originate from.
 
-1. Sign in to your Okta organization as a user with [administrator privileges](https://help.okta.com/okta_help.htm?id=ext_Security_Administrators).
+> **Note:** Record the token value. This is the only opportunity to see it and record it.
 
-     API tokens have the same permissions as the user who creates them, and if the user permissions change, the API token permissions also change.
+## Token network restrictions
 
-    See the section above on **Privilege level**, regarding the use of a service account when creating an API token, to specifically control the privilege level associated with the token.
+Administrators can specify a network range for every SSWS API token. The tokens only work if API requests are made from the specified IP or network range. You can specify network zones while you create a new API token. You can also modify an existing token to specify a network range.
```

Comment:
```suggestion
Admins can specify a network range for every SSWS API token. The tokens only work if API requests are made from the specified IP or network range. You can specify network zones while you create a new API token. You can also modify an existing token to specify a network range.
```

Reply:
The first sentence uses "Admins can", and the third sentence uses "You can". Is the audience the same for both?